README.md

```diff
@@ -249,6 +256,22 @@ postgresql_database 'vacuum databases' do
 end
 ```
 
+```ruby
+# Create, Insert, Query a SQLite database
+# Note that inserting anything in to the database will create it automaticly.
+sqlite_database 'mr_softie' do
+  connection '/path/to/database.db3'
```

Comment:
Can we make this database so it matches the postgresql provider and better describes what the attribute is?